src/libs/Navigation/FreezeWrapper.tsx

```diff
@@ -28,8 +27,9 @@ function FreezeWrapper({keepVisible = false, children}: FreezeWrapperProps) {
             // if the screen is more than 1 screen away from the current screen, freeze it,
             // we don't want to freeze the screen if it's the previous screen because the freeze placeholder
             // would be visible at the beginning of the back animation then
-            if ((navigation.getState()?.index ?? 0) - (screenIndexRef.current ?? 0) > 1) {
-                InteractionManager.runAfterInteractions(() => setIsScreenBlurred(true));
```

Comment:
> InteractionManager.runAfterInteractions

@getusha can you please check if this removal is fine?
I see you added this workaround during RNW upgrade [here](https://github.com/Expensify/App/commit/0b4ee5c396a14b07a464b70f47e696f5095e2c58). To fix what bug?

Reply:
answered [here](https://expensify.slack.com/archives/C05LX9D6E07/p1710172332635789?thread_ts=1710170982.916159&cid=C05LX9D6E07)